content/en/service_management/workflows/guide/private_action_credentials.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Handling Private Action Credentials
+kind: guide
+disable_toc: false
+---
+
+{{< callout url="https://google.com" btn_hidden="false" header="Join the Beta!">}}
+Private Actions are in beta. Use this form to request access today.
+{{< /callout >}}
+
+## Overview
+
+Some private actions such as Jenkins and PostgreSQL require credentials to function.
+- Create a JSON file for the credential and use the JSON structure provided in [Credential files](#credential-files).
+- Store your credential files in the configuration directory you created during setup.
+- When you specify the path to the credential in your runner connection, use the path to the credential file on the container.
+
+## Credential files
+
+{{< tabs >}}
+{{% tab "PostgreSQL" %}}
+
+The PostgreSQL connection requires a PostgreSQL Connection URI credential. 
+
+This example uses the URI: `postgres://usr:password@example_host:5432/example_db`. For information on constructing a PostgreSQL connection URI, see [the official PostgreSQL documentation][101].
+
+{{< code-block lang="json" filename="/etc/dd-action-runner/creds/creds.pgpass" disable_copy="false" collapsible="true" >}}
+{
+        "auth_type": "Token Auth",
+        "credentials": [
+                {
+                        "tokenName": "connectionUri",
+                        "tokenValue": "postgres://usr:password@example_host:5432/example_db"
+                }
+        ]
+}
+{{< /code-block >}}
+
+In the runner's connection, specify the location of the credential file on the private action runner container. In this example, the credential is stored at `/etc/dd-action-runner/creds/creds.pgpass` on the runner.
```

Comment:
The runner's connection is the connection that the runner has been added to. I've added the same preamble to the overview that exists in the other pages and linked off to the other docs, because the user should definitely look at that page before they try to set up credentials.

Reply:
On the runner means on the private action runner container.